tests/test_bukuDb.py

```diff
@@ -95,21 +115,21 @@ def setUp(self):
         os.environ["XDG_DATA_HOME"] = TEST_TEMP_DIR_PATH
 
         # start every test from a clean state
-        if exists(TEST_TEMP_DBFILE_PATH):
-            os.remove(TEST_TEMP_DBFILE_PATH)
+        rmdb()
 
         self.bookmarks = TEST_BOOKMARKS
         self.bdb = BukuDb()
 
     def tearDown(self):
         os.environ["XDG_DATA_HOME"] = TEST_TEMP_DIR_PATH
+        rmdb(self.bdb)
 
     @pytest.mark.non_tox
     def test_get_default_dbdir(self):
         dbdir_expected = TEST_TEMP_DBDIR_PATH
-        dbdir_local_expected = os.path.join(
-            os.path.expanduser("~"), ".local", "share", "buku"
-        )
+        home = os.path.expanduser("~")
+        dbdir_local_expected = (os.path.join(home, ".local", "share", "buku") if sys.platform != 'win32' else
+                                os.path.join(home, "AppData", "Roaming", "buku"))
```

Comment:
On Windows, the default folder for application data is `C:\%USERPROFILE%\AppData\Roaming\`